crypto/key.go

```diff
@@ -55,6 +102,24 @@ func StringHexToPub(suite abstract.Suite, s string) (abstract.Point, error) {
 	return point, err
 }
 
+// PubToString64 converts a Public point to a base64 representation
+func PubToString64(suite abstract.Suite, point abstract.Point) (string, error) {
+	pbuf, err := point.MarshalBinary()
+	return base64.StdEncoding.EncodeToString(pbuf), err
+}
+
+// String64ToPub reads a base64 representation of a public point and converts it
+// back to a point.
+func String64ToPub(suite abstract.Suite, s string) (abstract.Point, error) {
+	encoded, err := base64.StdEncoding.DecodeString(s)
+	if err != nil {
+		return nil, err
+	}
+	point := suite.Point()
+	err = point.UnmarshalBinary(encoded)
+	return point, err
```

Comment:
I think it can be reduced to one line with https://golang.org/pkg/strings/#NewReader
```
return Read64Pub(suite,strings.NewReader(s))
```
:)